src/snowflake/cli/plugins/nativeapp/artifacts.py

```diff
@@ -456,7 +452,9 @@ def build_bundle(
     for (absolute_src, absolute_dest) in bundle_map.all_mappings(
         absolute=True, expand_directories=False
     ):
-        symlink_or_copy(absolute_src, absolute_dest, deploy_root=deploy_root)
+        copy_artifacts_to_deploy_root(
```

Comment:
There is a choice here:
1. `bundle_map.all_mappings(..., expand_directories=True) which will ensure that `absolute_src` and `absolute_dest` are files only, and there `copy_artifacts_to_deploy_root` will only copy files.
2. Leave `bundle_map.all_mappings` as is, and therefore `absolute_src` can be a directory, and we rely on SecurePath.copy` from `copy_artifacts_to_deploy_root` to honor its contract to always perform a recursive copy.

Inviting opinions from the team here. Either way, we need to remove the symlinking, reasons described in the preamble.